lib/test/cdo/test_github.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../test_helper'
+require 'uri'
+require 'cdo/github'
+
+class GitHubTest < Minitest::Test
```

Comment:
Not much of great value here, but we didn't have any tests so I wanted to get something started. My though is that by overriding REPO, we catch regressions where tested methods hardcode `code-dot-org/code-dot-org` instead of using the constant. By running a `URI.parse` we catch regressions where a typo makes url's invalid. The final assertion of the exact URL admittedly makes the prior checks a bit redundant